src/TCommandLine.cpp

```diff
@@ -105,6 +105,14 @@ bool TCommandLine::event( QEvent * event )
     if( event->type() == QEvent::KeyPress )
     {
         QKeyEvent *ke = static_cast<QKeyEvent *>( event );
+
+        // Shortcut for keypad keys
+        if(((int)ke->modifiers() & 0x20000000) && mpKeyUnit->processDataStream( ke->key(), (int)ke->modifiers() ) )
```

Comment:
I'd suggest you don't use a numeric constant:
```
...((int)ke->modifiers() & 0x20000000)
```
use the proper flag:
```
( ke->modifiers() & Qt::KeypadModifier )
```
Also, is this the right place for it, what impact does it have on the keypad enter key handling, and does it work as intended on all platforms (what about Macs = don't they do odd things with the arrow keys in regard to being numpad keys)?